Documentation/building/code-coverage.md

```diff
@@ -60,3 +60,18 @@ The results for this one library will then also show up in the aforementioned in
 And then once the run completes:
     
     ..\..\..\bin\tests\coverage\index.htm
+
+## Code coverage with mscorlib code
+
+Some code in the corefx contracts does not live in the corefx repo, e.g. System.Runtime, System.Text.Encoding. To run coverage reports for these projects, you need to build mscorlib locally from the coreclr repo.
```

Comment:
I might phrase this first sentence slightly differently, just to be very specific about what we mean, e.g. "Some of the libraries for which contracts and tests live in the corefx repo are actually implemented in the core runtime library in another repo, e.g. the implementation that backs the System.Runtime contract is in System.Private.Corlib.dll in either the coreclr or corert repo."